src/main/java/org/jenkinsci/plugins/durabletask/executors/ContinuedTask.java

```diff
@@ -60,6 +76,10 @@ private static boolean isContinued(Queue.Task task) {
             if (isContinued(item.task)) {
                 return null;
             }
+            Jenkins jenkins = Jenkins.getInstance();
+            if (jenkins == null || jenkins.getInitLevel().compareTo(InitMilestone.COMPLETED) < 0) {
```

Comment:
I am not comfortable with this plugin modifying a subtle aspect of queue processing in Jenkins even when no existing tasks implement the interface. If there is something wrong in base Jenkins then a bug should be filed for it and it should be fixed in core.